reconcile/openshift_namespaces.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+import reconcile.openshift_resources as openshift_resources
+
+import utils.gql as gql
+
+from utils.openshift_resource import ResourceInventory
+from utils.oc import StatusCodeError
+
+from multiprocessing.dummy import Pool as ThreadPool
+from functools import partial
+
+QUERY = """
+{
+  namespaces: namespaces_v1 {
+    name
+    cluster {
+      name
+      serverUrl
+      jumpHost {
+          hostname
+          knownHosts
+          user
+          port
+          identity {
+              path
+              field
+              format
+          }
+      }
+      automationToken {
+        path
+        field
+        format
+      }
+    }
+  }
+}
+"""
+
+
+def get_desired_state():
+    gqlapi = gql.get_api()
+    query = gqlapi.query(QUERY)['namespaces']
+    for namespace_info in query:
+        # adjust to match openshift_resources functions
+        namespace_info['managedResourceTypes'] = ['Namespace']
```

Comment:
how about instead of doing this, which is a bit ugly, we make it a kwarg for init_specs_to_fetch (defaulting to `["managedResourcesTypes"]`)

Reply:
also by the way, the proper way to create the default value is inside the method, not in the method's signature

Reply:
fixed